packages/node_modules/@webex/plugin-meetings/test/unit/spec/members/index.js

```diff
@@ -212,5 +212,33 @@ describe('plugin-meetings', () => {
         assert.isRejected(members.raiseOrLowerHand('test1', true));
       });
     });
+
+    describe('#lowerAllHands', () => {
+      it('should fire spies correctly when lowerAllHands is called with valid params', async () => {
+        const members = createMembers({url: url1});
+
+        const {membersRequest} = members;
+
+        const generateOptionsSpy = sandbox.spy(MembersUtil, 'generateLowerAllHandsMemberOptions');
+        const lowerAllHandsMemberSpy = sandbox.spy(membersRequest, 'lowerAllHandsMember');
+
+        await members.lowerAllHands('test1');
+
+        assert.calledOnce(generateOptionsSpy);
+        assert.calledOnce(lowerAllHandsMemberSpy);
+      });
+
+      it('should throw a rejection if there is no locus url', async () => {
+        const members = createMembers({url: false});
+
+        assert.isRejected(members.lowerAllHands('test1'));
+      });
+
+      it('should throw a rejection if there is no requestingMemberId', async () => {
+        const members = createMembers({url: url1});
+
+        assert.isRejected(members.lowerAllHands(undefined));
+      });
```

Comment:
suggestion: A check to ensure a request is not attempted in these cases would also be good (to ensure the rejection is from here).